src/components/CCPDashboard/CCPDashboardPage.tsx

```diff
@@ -118,6 +160,81 @@ const CCPDashboardPage = ({ match }: RouteComponentProps<TParams>) => {
     [allPatients, ccpId]
   );
 
+  const highlightPatient = (id) => {
+    clearTimeout(lastUpdatedPatientTimeout);
+    setLastUpdatedPatient(id);
+    const highlightTimeout = setTimeout(() => {
+      setLastUpdatedPatient('');
+    }, 5000);
+    setLastUpdatedPatientTimeout(highlightTimeout);
+  };
+
+  React.useEffect(() => {
+    highlightPatient(userUpdatedPatientId);
+  }, [userUpdatedPatientId]);
+
+  useSubscription(PATIENT_UPDATED, {
+    variables: { eventId },
+    onSubscriptionData: ({ client, subscriptionData: { data } }) => {
+      highlightPatient(data.patientUpdated.id);
+      client.writeFragment({
+        id: `Patient:${data.patientUpdated.id}`,
+        fragment: gql`
+          fragment NewPatient on Patient {
+            gender
+            age
+            runNumber
+            barcodeValue
+            triageLevel
+            status
+            notes
+            updatedAt
+            transportTime
+            hospitalId {
+              id
+              name
+            }
+            ambulanceId {
+              id
+              vehicleNumber
+            }
+          }
+        `,
```

Comment:
Just curious, is there a reason why this isn't imported from a separate file like all of our other queries/mutations?

Reply:
Patient details dialog should not update in real time, the component was just refactored to minimize graphql calls upon opening from a row. I think the fragment was not imported because it was a fragment instead of a query, but I think we can import it.

Reply:
updated code so patientdetailsdialog now updates in real time